File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1180.py
n = int(input())
nums = [int(i) for i in input().split()]
menor = nums[0]
pos=1

for i in range(1, n):
    if(nums[i]<menor):
        menor = nums[i]
        pos = i
print("Menor valor: "+str(menor))
print("Posicao: "+str(pos))

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1180.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

n = parseInt(lines.shift())
nums = lines.shift().split(' ')
menor = parseInt(nums[0])
pos=1

for(i=1;i<n;i++){
    if(parseInt(nums[i])<menor){
        menor = parseInt(nums[i])
        pos=i
    }
}
console.log("Menor valor: "+ menor);
console.log("Posicao: "+pos);